src/main/java/uk/nhs/digital/nhsconnect/nhais/model/jsonpatch/JsonPatches.java

```diff
@@ -8,100 +9,107 @@
 @RequiredArgsConstructor
 public class JsonPatches {
 
-    private static final String TITLE_PATH = "/name/0/prefix/0";
-    private static final String SURNAME_PATH = "/name/0/family/";
-    private static final String PREVIOUS_SURNAME_PATH = "/name/1/family/";
-    private static final String FIRST_FORENAME_PATH = "/name/0/given/0";
-    private static final String SECOND_FORENAME_PATH = "/name/0/given/1";
-    private static final String OTHER_FORENAMES_PATH = "/name/0/given/2";
-    private static final String SEX_PATH = "/gender";
-    private static final String BIRTH_DATE_PATH = "/birthDate";
-    private static final String HOUSE_NAME_PATH = "/address/0/line/0";
-    private static final String HOUSE_NUMBER_PATH = "/address/0/line/1";
-    private static final String POST_TOWN_PATH = "/address/0/line/2";
-    private static final String COUNTY_PATH = "/address/0/line/3";
-    private static final String POSTAL_CODE_PATH = "/address/0/postalCode";
-
+    public static final String ALL_FORENAMES_PATH = "/name/0/given";
+    public static final String TITLE_PATH = "/name/0/prefix/0";
+    public static final String SURNAME_PATH = "/name/0/family";
+    public static final String PREVIOUS_SURNAME_PATH = "/name/1/family";
+    public static final String FIRST_FORENAME_PATH = "/name/0/given/0";
+    public static final String SECOND_FORENAME_PATH = "/name/0/given/1";
+    public static final String OTHER_FORENAMES_PATH = "/name/0/given/2";
+    public static final String SEX_PATH = "/gender";
+    public static final String BIRTH_DATE_PATH = "/birthDate";
+    public static final String HOUSE_NAME_PATH = "/address/0/line/0";
+    public static final String HOUSE_NUMBER_PATH = "/address/0/line/1";
+    public static final String POST_TOWN_PATH = "/address/0/line/2";
+    public static final String COUNTY_PATH = "/address/0/line/3";
+    public static final String POSTAL_CODE_PATH = "/address/0/postalCode";
+
+    @Getter
+    private final AmendmentBody amendmentBody;
```

Comment:
Is it necessary? I don't like cyclic dependencies.

Reply:
You're right. I though that it's needed to not have to pass all other required fields but in fact it's the interface I've created that is wrong and should accept whole `AmendmentBody` instead of just the patches. I'll change that